### PR TITLE
Boards: Add da14695 dk usb

### PR DESCRIPTION
**Pull request by NiekIlmer**
This PR adds support for the DA14695 USB kit

Three commits in this PR:
-add UART2 and 3 DT bindings for DA1469x
-add the DA14695 variant thats used on this board
-add board files for da14695_dk_usb

### CODE REVIEW

**Comment by MaureenHelm** [comment]
@andrzej-kaczmarek please take a look
@NiekIlmer can you fix the failed CI check please?

File: boards/arm/da14695_dk_usb/da14695_dk_usb-pinctrl.dtsi [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 Renesas Electronics Corporation and/or its affiliates
```
Comment (andrzej-kaczmarek) [inline boards/arm/da14695_dk_usb/da14695_dk_usb-pinctrl.dtsi:2]: 2023?

also any reason why there are 2 different copyrights lines used in all files?

**Review by andrzej-kaczmarek — CHANGES_REQUESTED** [review]
code looks good, but please fix commits order to match dependencies (i.e. board depends on SoC, not the other way round) and make prefixes in commit titles lowercase to match existing commits

File: boards/arm/da14695_dk_usb/da14695_dk_usb-pinctrl.dtsi (hunk 1 above)
Comment (NiekIlmer) [reply boards/arm/da14695_dk_usb/da14695_dk_usb-pinctrl.dtsi]: This was fixed. All files now have the same copyright line and the right year.

**Review by andrzej-kaczmarek — APPROVED** [review]

**Review by aaillet — APPROVED** [review]
LGTM
Have you run the twister test suite and basic tests on real hardware?

**Pull request merged**